wbsv/Find.py

```diff
@@ -46,6 +49,20 @@ def remove_useless(l):
         """Remove not available data from list."""
         return {x for x in l if x is not None and len(x) > 1}
 
+    def fetch_url(self, url):
+        """
+        Abstract method
+
+        @args:
+            url: URL
+        @return:
+            List of URLs included on document of `url`
+        """
+        return {}
+
+    def wait(self):
+        time.sleep(randint(2, 5))
```

Comment:
This parameter seem too long...
I'd like to be shorter or let users set it up as they wish.

Reply:
OK, I manage to accept its latency...